### ISSUE TITLE
collective.faq does not show up in Add-ons controlpanel

### ISSUE THREAD

**Issue by tisto** [collective/collective.faq#3]
Plone 4.3.14 with collective.faq 1.0.0a2.

buildout.cfg:

```
[buildout]
extends = https://dist.plone.org/release/4.3.14/versions.cfg
parts = instance

[instance]
recipe = plone.recipe.zope2instance
http-address = 8080
user = admin:admin
eggs =
    Plone
    plone.restapi
    collective.faq

[versions]
setuptools =
zc.buildout =
```

Steps to reproduce:

1) Run buildout
2) Start instance
3) Create Plone instance -> c.faq does not show up
4) Go to add-ons controlpanel
-> c.faq does not show up

**Comment by rodfersou** [comment]
This bug is caused because there is no `configure.zcml` inside the package:
```bash
 ...collective.faq-1.0.0a3-py2.7.egg/collective/faq 
$ pwd
/home/rodfersou/.projects/cache/eggs/collective.faq-1.0.0a3-py2.7.egg/collective/faq

 ...collective.faq-1.0.0a3-py2.7.egg/collective/faq 
$ ls
browser     content.pyc  __init__.py   __init__.pyo   interfaces.pyc  setuphandlers.py   setuphandlers.pyo  testing.pyc  tests        upgrades.pyc
content.py  content.pyo  __init__.pyc  interfaces.py  interfaces.pyo  setuphandlers.pyc  testing.py         testing.pyo  upgrades.py  upgrades.pyo
```

So I imagine there is some problem in the package generation.

**Comment by tisto** [comment]
Fixed with 1.0.0a4 release.
